website/templates/emails/reviews_submission_confirmation.html.mako

```diff
@@ -9,86 +9,99 @@
-                                                                    <img src="${'{}static/img/fa-facebook-blue.png'.format(settings.DOMAIN)}" alt="facebook" style="display: block; border: 0;outline: none;text-decoration: none; text-align: center;vertical-align: bottom;" width="14">
-                                                                </a>
-                                                            </td>
-                                                            <td>
-                                                                <a href="${u'https://www.linkedin.com/shareArticle?mini=true&url={link}&summary=Read%20my%20{word}%2C%20%E2%80%9C{title}%E2%80%9D%20on%20{name}%20{link}&title=I%20just%20posted%20a%20{word}&source='.format(word=document_type,title=reviewable.title, name=reviewable.provider.name, link=reviewable.absolute_url)}" target="_blank">
-                                                                    <img src="${'{}static/img/fa-linkedin-blue.png'.format(settings.DOMAIN)}" alt="LinkedIn" style="display: block; border: 0;outline: none;text-decoration: none; text-align: center;vertical-align: bottom;" width="14">
-                                                                </a>
-                                                            </td>
-                                                        </tr>
-                                                    </tbody>
-                                                </table>
-                                            </li>
-                                        </ul>
-                                    </td>
-                                </tr>
-                            </tbody>
-                        </table>
                     % endif
-                    % if not no_future_emails and not isOsfSubmission:
-                        You will receive a separate notification informing you of any status changes.
-                    % endif
-                </p>
-                % if not is_creator:
-                <p>
-                    If you have been erroneously associated with "${reviewable.title}," then you may visit the ${document_type}
-                    and remove yourself as a contributor.
-                </p>
-                % endif
-                <p>Learn more about <a href="${provider_url}">${reviewable.provider.name}</a> or <a href="https://osf.io/">OSF</a>.</p>
+                    <p>Learn more about <a href="${provider_url}">${reviewable.provider.name}</a> or <a href="https://osf.io/">OSF</a>.</p>
+                    <br>
+                    <p>
+                        Sincerely,<br>
+                        ${'The OSF team' if isOsfSubmission else 'The {provider} and OSF teams'.format(provider=reviewable.provider.name)}
+                    </p>
+                </div>
+            % else:
+                Hello ${user.fullname},
+                <br>
```

Comment:
missing a paragraph, I believe:

"Your ${document_type} <a href="${reviewable.absolute_url}">${reviewable.title}</a> has been successfully submitted to ${reviewable.provider.name}."